src/android/io/branch/BranchSDK.java

```diff
@@ -231,13 +232,7 @@ private void initSession(CallbackContext callbackContext)
         this.activity = this.cordova.getActivity();
 
         this.instance = Branch.getAutoInstance(this.activity.getApplicationContext());
-
-        if (callbackContext == null) {
-            this.instance.initSession(new SessionListener(), activity.getIntent().getData(), activity);            
-        } else {
-            this.instance.initSession(new SessionListener(callbackContext), activity.getIntent().getData(), activity);
-        }
-
+        this.instance.initSession(new SessionListener(callbackContext), activity.getIntent().getData(), activity);
```

Comment:
The old code seems verbose. You can just pass in null for the callbackContext and this should achieve the same effect.